Ports/RISC-V/RV32/GCC/os_cpu_a.S

```diff
@@ -82,6 +84,8 @@ OSStartHighRdy:
     # mstatus = mstatus & ~t0
     csrrc  zero, mstatus, t0
 
+    # setup the isr stack pointer
+    jal     OSInitHook
```

Comment:
This gets called in external/ucos/uC-OS3/Source/os_core.c, OSInit so its not needed here